src/elan/json.rs

```diff
@@ -0,0 +1,65 @@
+use Cfg;
+use std::path::PathBuf;
+use std::io::Result;
+
+use serde_derive::Serialize;
+
+use crate::OverrideReason;
+
+#[derive(Serialize)]
+struct Version {
+    current: String,
+    /// `None` on network error
+    newest: Option<String>,
```

Comment:
Would it be possible to provide more information in the case of a network error?

Reply:
I suppose a single string would be possible